src/components/dialogs/parameters/sensi/sensitivity-Flow-parameters.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import { Grid } from '@mui/material';
+
+import {
+    ANGLE_FLOW_SENSITIVITY_VALUE_THRESHOLD,
+    FLOW_FLOW_SENSITIVITY_VALUE_THRESHOLD,
+    FLOW_VOLTAGE_SENSITIVITY_VALUE_THRESHOLD,
+} from '../../../utils/field-constants';
+import { FormattedMessage } from 'react-intl';
+import { useStyles } from '../parameters';
+import { FloatInput } from '@gridsuite/commons-ui';
+
+const SensitivityAnalysisFields = () => {
+    const classes = useStyles();
+
+    const flowFlowSensitivityValueThresholdField = (
```

Comment:
We can make an array that contains FloatInputNames and make a map of this array to list FloatInput. The 3 FloatInput are the same except the name.

Reply:
done